### ISSUE TITLE
Question about reproduced mAP

### ISSUE THREAD

**Issue by geonuk-kimmm** [HeartFu/RelationNetworkForObjectDetection-Pytorch#1]
Thanks for your codes.
I notice that you got mAP 36% with current code on COCO2014.
Did you use minival as test set?

**Comment by HeartFu** [comment]
No, I used original train and val set.
